src/components/HashButton.tsx

```diff
@@ -57,9 +57,10 @@ function HashLink(hash: string, type: HashType): JSX.Element {
 interface HashButtonProps {
   hash: string;
   type: HashType;
+  alignRight?: boolean;
 }
 
-export default function HashButton({hash, type}: HashButtonProps) {
+export default function HashButton({hash, type, alignRight}: HashButtonProps) {
```

Comment:
I think it would be better to use spread props here (that way it can handle more than just aligning right).